debugging.md

```diff
@@ -25,13 +25,13 @@ Auch eine REDAXO-Installation kann einmal „Schluckauf“ haben. Im Folgenden w
 <a name="ooops"></a>
 ## Fehleranalyse: Whooops und Oooops
 
-![Ooops](/assets/v5.10.0-debug_ooops.png)
+![Oooops](/assets/v5.10.0-debug_ooops.png) Allgemeine Oooops Fehlerseite - Dargestellt im Frontend oder Backend
 
 Tritt ein Fehler auf, meldet REDAXO sich im Frontend mit einem Oooops und im Backend mit einem Rrrrroar.
 
-![Whoops](/assets/v5.10.0-debug_whooops.png)
+![Whooops](/assets/v5.10.0-debug_whooops.png) Whooops Fehlerseite mit Debuginformationen
 
-Wenn ein Administrator eingeloggt ist, oder der Administrator den  Debug-Modus aktiviert hat, wird anstelle des Oooops ein Whoooops mit genauerer Fehlerbeschreibung und Stacktrace ausgegeben.
+Wenn ein Administrator eingeloggt ist, oder der Administrator den Debug-Modus aktiviert hat, wird anstelle des Oooops ein Whooops mit genauerer Fehlerbeschreibung und Stacktrace ausgegeben um die Fehlersuche zu vereinfachen

```

Comment:
```suggestion
Wenn ein Administrator eingeloggt ist, oder der Administrator den Debug-Modus aktiviert hat, wird anstelle des Ooops ein Whooops mit genauerer Fehlerbeschreibung und Stacktrace ausgegeben, um die Fehlersuche zu vereinfachen.

```

Reply:
Die anzhal der ooo in Ooooops und Whooops war an vielen stellen unterschiedlich.. ich hab angefangen mit meinen Änderungen zu vereinheitlichen.. hier hast du ein Ooops vorgeschlagen... mir ist es vom Prinzip egal wie wir es schreiben,.. baer einheitlich fänd ich wichtig